operations/rates.md

```diff
@@ -104,6 +104,123 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `AvailabilityBlockAssignments` | array of [Availability block assignment](#availability-block-assignment) | optional | Shows which rates relate to which availability blocks. |
 | `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
 
+## Add rates
+
+Adds rates the enterprise. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/rates/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+    "Rates":[
+      {
+          "Names": { "EN": "My rate" },
```

Comment:
Should this be "Names" (plural) or "Name" (singular)? Also, shouldn't the language code be "en-US" not "EN"?

Reply:
Should be plural since it's dictionary. The language code is coming from business logic/database.